.github/workflows/dav1d.yml

```diff
@@ -1,45 +1,38 @@
 name: dav1d
-
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: '0 6 * * 1'
```

Comment:
Why a weekly run?

Please add a comment close to this line or above it: `# Run every Monday at 06:00 (UTC)`

Reply:
I added the comment!

Running the CI periodically ensures errors by environment or dependencies updates, such as Nasm, Meson, Ninja or GCC are noticed within a certain timespan. It also could help against catching error by updates to dav1d itself. This way it also makes it easier to distinct errors from code changes, since the environment is tested periodically and can be ruled out.

Reply:
Perfect, now I see your motivation. Thanks for your explanation!

Reply:
You're welcome, thanks for merging! :)